common/src/main/java/bisq/common/persistence/PersistenceManager.java

```diff
@@ -166,6 +183,7 @@ private static void onWriteCompleted(ResultHandler completeHandler,
     @Nullable
     private Timer timer;
     private ExecutorService writeToDiskExecutor;
+    public final AtomicBoolean initCalled = new AtomicBoolean(false);
```

Comment:
`initCalled` is always false as it's only set here

Reply:
Ups, thanks for spotting that...